### ISSUE TITLE
Make a separate method for calculating the accuracy 🎯

### ISSUE THREAD

**Issue by maksymalist** [maksymalist/rust-tensorflow-mini#2]
`file path: examples/main.rs`
<img width="813" alt="Screenshot 2023-06-25 at 2 24 50 PM" src="https://github.com/maksymalist/rust-tensorflow-mini/assets/79988159/0525de84-1d49-4d52-bb63-12f0ea6d4b78">

Make the loss function only calculate the loss and make a separate file / module to calculate the accuracy
